pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -346,6 +346,8 @@ type ProviderOpts struct {
 	// Use an instance template and a managed instance group to create VMs. This
 	// enables cluster resizing, load balancing, and health monitoring.
 	Managed bool
+	// Specify spot instance zones for the managed instance group.
+	ManagedSpotZones []string
```

Comment:
We could also say that this is supposed to be a subset of `Zones` and that VMs running on zones not in this list will be provisioned on-demand.

Reply:
Updated, made it more descriptive.